src/app/core/interfaces/widgets-interface.ts

```diff
@@ -266,6 +266,32 @@ export interface IWidgetSvcConfig {
   nextDashboard?: number;
   /** If true, play beeps when the racer-timer-widget counts to through the minutes, 10s and each of the last 10s. */
   playBeeps?: boolean;
+
+  /** Used by history chart widget: URL of the history API endpoint */
+  historyApiUrl?: string;
+  /** Used by history chart widget: Array of Signal K paths to chart */
+  historyPaths?: string[];
+  /** Used by history chart widget: Array of aggregation methods to use */
+  aggregationMethods?: string[];
+  /** Used by history chart widget: Time selection mode ('range' | 'duration') */
+  timeMode?: string;
+  /** Used by history chart widget: Start time (ISO string or 'now') */
+  startTime?: string;
+  /** Used by history chart widget: End time (ISO string) */
+  endTime?: string;
+  /** Used by history chart widget: Duration value for duration mode */
+  duration?: number;
+  /** Used by history chart widget: Duration unit ('s', 'm', 'h', 'd') */
```

Comment:
The api spec says iso 8601 durations, should the values here use the same?

https://docs.digi.com/resources/documentation/digidocs/90001488-13/reference/r_iso_8601_duration_format.htm